includes/KSES.php

```diff
@@ -540,13 +540,22 @@ public function filter_kses_allowed_html( $allowed_tags ) {
 	public function filter_content_save_pre_before_kses( $post_content ) {
 		return (string) preg_replace_callback(
 			'|(?P<before><\w+(?:-\w+)*\s[^>]*?)style=\\\"(?P<styles>[^"]*)\\\"(?P<after>([^>]+?)*>)|', // Extra slashes appear here because $post_content is pre-slashed..
-			static function ( $matches ) {
-				return $matches['before'] . sprintf( ' data-temp-style="%s" ', $matches['styles'] ) . $matches['after'];
-			},
+			[ $this, 'filter_content_save_pre_before_kses_matches' ],
```

Comment:
I think we should keep the anonymous function here.
WP core standards allow anonymous functions for `preg_replace_callback` too.

For testing, we can just test `filter_content_save_pre_before_kses` itself.

Reply:
Agreed. I also noted it might be overkill. I will revert.

Reply:
Fixed https://github.com/google/web-stories-wp/pull/2593/commits/5d6850084b0c4f84470e3c48977830551d6bf073